tools/test_util.py

```diff
@@ -114,6 +114,8 @@ def filter_test_suite(suite, pattern):
 
 
 def run_tests(test_cases=None):
+    os.environ["DENO_BUILD_PATH"] = build_path()
```

Comment:
is this necessary?

Reply:
I need some way to locate the built version of `test_plugin`(see `//tests/033_native_plugins.ts`).

Reply:
Can you just write the path explicitly? Overwriting build path env seems unnecessary/destructive.

Does the plugin need to be built to the same release/debug mode as deno itself?

Reply:
This won't have any affect if `DENO_BUILD_PATH` is already set.
``` python
def build_path():
    if "DENO_BUILD_PATH" in os.environ:
        return os.environ["DENO_BUILD_PATH"]
    else:
        return os.path.join(root_path, "target", build_mode())
```

Reply:
What about writing the path explicitly? I think that would be preferable...

Reply:
What do you mean by explicit? We need to provide an absolute path, and that path is going to be different for release/debug builds.

Reply:
related, this is also a little awkward in the ts file:

     const dLib = dlopen(env().DL_PATH_TEST_BINDING + "/" + dlname("test_plugin")); 

---

In the tests it seems like (in travis) you are only building release (I may be misreading that), so by "explicit" I mean link to the release plugin build...

Or is the issue in a debug deno you can't link to a release plugin?